extensions/src/AWSSDK.Extensions.Configuration.AWSSystemsManager/AWSSDK.Extensions.Configuration.AWSSystemsManager.nuspec

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package> 
+  <metadata> 
+    <id>AWSSDK.Extensions.Configuration.AWSSystemsManager</id>
+    <title>AWSSDK - ASP.NET Configuration Extensions for AWS Systems Manager</title>
+    <version>3.3.0.0</version>
+    <authors>Amazon Web Services, Ken Hundley</authors>
+    <description>ASP.NET Configuration Extensions for AWS Systems Manager</description>
+    <language>en-US</language>
+    <licenseUrl>http://aws.amazon.com/apache2.0/</licenseUrl>
+    <projectUrl>https://github.com/aws/aws-sdk-net/</projectUrl>
+    <tags>AWS Amazon aws-sdk-v3</tags>
+    <iconUrl>https://sdk-for-net.amazonwebservices.com/images/AWSLogo128x128.png</iconUrl>
+	
+    <dependencies>
+      <group targetFramework="netstandard1.3">
```

Comment:
Again I think we should start the baseline at 2.0

Reply:
Done